static/src/stylesheets/base/_zindex.scss

```diff
@@ -8,3 +8,4 @@ $zindex-sticky: 1020;  // Sticky header
 $zindex-modal: 1030;   // Search modal
 $zindex-popover: 1040; // Breaking news
 $zindex-overlay: 1050; // Lightbox
+$zindex-notifications-permissions-warning: 15000;
```

Comment:
Is there a z-index somewhere else that caused you to use `15000`? If not, and you're just trying to up a level (I think you just want this to appear above everything), then `1060` should work here.